src/components/Dashboard/Form/DocumentForm.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from "react";
+import FormHeader from "./FormHeader/FormHeader.js";
+import { Form } from "react-final-form";
+import PropTypes from "prop-types";
+
+export default class DocumentForm extends Component {
+  state = {
+    page: 0,
+    values: this.props.initialValues || {}
```

Comment:
Initialize a state by `props` is in general antipattern and we should try to avoid this. I recommend usage `componentWillReceiveProps` like in this article (point 4) https://codeburst.io/how-to-not-react-common-anti-patterns-and-gotchas-in-react-40141fe0dcd

Reply:
As React dosc says:
> These methods are considered legacy and you should avoid them in new code:
UNSAFE_componentWillUpdate()
UNSAFE_componentWillReceiveProps()

so instead method shouldComponentUpdate will be added later on ;)

Reply:
Yes, my bad. I've thought about `shouldComponentUpdate`. :+1: